test/basic.spec.js

```diff
@@ -29,6 +30,18 @@ describe('too-wordy', () => {
     });
   });
 
+  describe('a sentence with hyphen should parse word as one', () => {
```

Comment:
Should be rephrased: "hyphenated words should not be parsed as two words". I think this is more clear.